app/src/pages/market_sections/market_home_container.tsx

```diff
@@ -304,16 +341,18 @@ const MarketHomeContainer: React.FC = () => {
       const routeQueryStart = '?'
       const routeQueryArray: string[] = []
 
-      if (filter.sortBy === `sort24HourVolume${Math.floor(Date.now() / (1000 * 60 * 60)) % 24}`) {
-        route += '/24h-volume'
-      } else if (filter.sortBy === 'usdVolume') {
-        route += '/volume'
-      } else if (filter.sortBy === 'creationTimestamp') {
-        route += '/newest'
-      } else if (filter.sortBy === 'openingTimestamp') {
-        route += '/ending'
-      } else if (filter.sortBy === 'usdLiquidityParameter') {
-        route += '/liquidity'
+      if (filter.state !== 'MY_MARKETS') {
+        if (filter.sortBy === `sort24HourVolume${Math.floor(Date.now() / (1000 * 60 * 60)) % 24}`) {
+          route += '/24h-volume'
+        } else if (filter.sortBy === 'usdVolume') {
+          route += '/volume'
+        } else if (filter.sortBy === 'creationTimestamp') {
+          route += '/newest'
+        } else if (filter.sortBy === 'openingTimestamp') {
+          route += '/ending'
+        } else if (filter.sortBy === 'usdLiquidityParameter') {
+          route += '/liquidity'
+        }
```

Comment:
@KadenZipfel You are gonna kill me 🗡️  Should we also change it here...Should have outlined all at the start...

Reply:
Good catch! Will fix